site/app/views/forum/ForumThreadView.php

```diff
@@ -747,6 +747,10 @@ public function displayThreadList($threads, $filtering, &$activeThreadAnnounceme
                 $class .= " new_thread";
             }
             if ($thread["deleted"]) {
+                $array_of_class_IDs = explode(" ", $class);
+                if (in_array("new_thread", $array_of_class_IDs)) {
```

Comment:
It might be a bit better to use the actual variables that determines whether it's a new thread or not rather than the class list. So on line 746 you could just store that result and substitute that new variable in here and on 746.